spec/lib/sm/api/message_drafts_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'sm/client'
+
+describe SM::Client do
+  let(:config) { SM::Configuration.new(attributes_for(:configuration)) }
+  let(:session) { SM::ClientSession.new(attributes_for(:session, :valid_user)) }
+  let(:user_id) { 10_616_687 }
+
+  before(:each) do
+    @client = SM::Client.new(config: config, session: session)
+  end
+
+  describe 'post_create_message_draft' do
+    let(:draft) { attributes_for(:message).except(:id) }
+
+    it 'creates a new draft without attachments' do
+      VCR.use_cassette("sm/message_drafts/#{user_id}/create") do
+        @client.authenticate
+        client_response = @client.post_create_message_draft(draft)
+
+        expect(client_response).to be_a(MessageDraft)
+      end
+    end
+
+    it 'updates an existing draft' do
+      VCR.use_cassette("sm/message_drafts/#{user_id}/update") do
+        @client.authenticate
+        msg = @client.post_create_message_draft(draft)
```

Comment:
this looks like you're recording setting up the VCR, you should use Postman to setup whatever state you need before doing delete or updates spec testing -- that way VCR is only concerned with recording that specific operation. Otherwise might as well wrap the entire spec in a single VCR call and only record once.